lib/vehicle_factory.rb

```diff
@@ -0,0 +1,42 @@
+require "spec_helper"
+
+class VehicleFactory
+  attr_reader :vehicles
+
+  def initialize
+    @vehicles = []
+  end
+
+  def create_vehicles(database)
+    if database.first.keys.include?(:scofflaw_indicator)
+      vehicle_database = database.find_all do |record|
+        record[:record_type] == "VEH"
+      end
+      # require 'pry'; binding.pry
```

Comment:
Watch out for lingering prys and comments on final submissions. We don't ever want those to accidentally get into main on a production app (especially if they aren't commented out).